rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -464,13 +465,7 @@ if [ -d "%{rudderdir}/jetty7/contexts" ]; then
   ln -sf %{rudderdir}/share/webapps/rudder.xml %{rudderdir}/jetty7/contexts/rudder.xml
 fi
 
-# Warn the user that Jetty needs restarting. This can't be done automatically due to a bug in Jetty's init script.
```

Comment:
it was most likely something in Jetty 7.2.2 where it didn't correctly finish/closed tty